tests/testthat/test-renderer1-tooltip.R

```diff
@@ -107,22 +108,13 @@ info <- animint2HTML(viz)
 
 test_that("tooltip works with href",{
   # Test for bug when points are not rendered with both href + tooltip
-  
-  # See that points are rendered for every country. This test may need to be
-  # changed soon!!! See the viz on the link below:
-  # http://bl.ocks.org/faizan-khan-iit/449b84b2e6b55ca5e39ea9aaad97e27e
-  # You may notice a point on the very top-left of the viz, outside the plot.
-  # These are the points that have NaN as their value in the DOM, arriving from
-  # NA values in the tsv. Should be an easy fix. But leaving a comment here
```

Comment:
@faizan-khan-iit this PR fixes that, by removing the NA data points from the tsv files.